lib/ReadConfig.pm

```diff
@@ -826,6 +828,57 @@ sub show_package_deps
 }
 
 
+# Read real version/product info from /etc/os-release.
+#
+# This only works _after_ we have setup the 'base' system in tmp/base.
+#
+sub get_version_info
+{
+  my $file = "${BasePath}tmp/base/etc/os-release";
+  my $f;
+  my %config;;
+
+  if(open $f, $file) {
+    while(<$f>) {
+      $config{$1} = $2 if /^([^\s=]+)\s*=\s*\"?(.*?)\"?\s*$/;
+    }
+  }
+  else {
+    return;
+  }
+
+  # build nice product name
+
+  my $product = $config{PRETTY_NAME};
+  $product =~ s/\s*\([^)]*\)$//;
```

Comment:
Ok, in this case, strip the '(ARCH)' post fix, if there's any - see sample file.